website/docs/api/themes/theme-configuration.md

```diff
@@ -246,35 +252,53 @@ module.exports = {
 };
 ```
 
-React Router should automatically apply active link styling to links, but you can use `activeBasePath` in edge cases. For cases in which a link should be active on several different paths (such as when you have multiple doc folders under the same sidebar), you can use `activeBaseRegex`. `activeBaseRegex` is a more flexible alternative to `activeBasePath` and takes precedence over it -- Docusaurus parses it into a regular expression that is tested against the current URL.
+### Navbar dropdown {#navbar-dropdown}
 
-Outbound (external) links automatically get `target="_blank" rel="noopener noreferrer"` attributes.
+Navbar items of the type `dropdown` has the additional `items` field, an inner array of navbar items. The dropdown items cannot be dropdowns themselves. In other words, they can of be one of the following types: `default`, `doc`, `docsVersion`. **This is henceforth referred to as link-like items.**
 
-### Navbar dropdown {#navbar-dropdown}
+Note that the dropdown base item is a link as well, so it is possible be clickable by itself and therefore receive any of the props of a [plain navbar link](#navbar-items).
 
-Navbar items can also be dropdown items by specifying the `items`, an inner array of navbar items.
+:::caution
 
-```js {9-19} title="docusaurus.config.js"
+The `dropdown` type used to be inferred from the existence of `items`. This behavior is deprecated and may stop working in the future. Always explicitly define the item to be of type `dropdown`.
```

Comment:
I think we can remove this for now.

If we decide to deprecate this we'll just add a Joi warning, but maybe it's more convenient to keep it for now